dynamodb/src/it/scala/zio/dynamodb/LiveSpec.scala

```diff
@@ -0,0 +1,589 @@
+package zio.dynamodb
+
+import com.amazonaws.services.dynamodbv2.local.server.DynamoDBProxyServer
+import io.github.vigoo.zioaws.core.config
+import software.amazon.awssdk.auth.credentials.SystemPropertyCredentialsProvider
+import zio.blocking.Blocking
+import zio.dynamodb.UpdateExpression.Action.SetAction
+import zio.dynamodb.UpdateExpression.SetOperand
+import zio.dynamodb.PartitionKeyExpression.PartitionKey
+import zio.dynamodb.SortKeyExpression.SortKey
+import io.github.vigoo.zioaws.{ dynamodb, http4s }
+import zio._
+import zio.dynamodb.DynamoDBQuery._
+import zio.dynamodb.ProjectionExpression._
+import zio.test.Assertion._
+import zio.test.environment._
+import software.amazon.awssdk.regions.Region
+import zio.schema.{ DeriveSchema, Schema }
+import zio.stream.ZSink
+import zio.test._
+import zio.test.TestAspect._
+
+import java.net.URI
+import scala.collection.immutable.{ Map => ScalaMap }
+
+object LiveSpec extends DefaultRunnableSpec {
+
+  val awsConfig = ZLayer.succeed(
```

Comment:
```suggestion
  private val awsConfig = ZLayer.succeed(
```
can these vals be made private?